server_code/historic_server/persistence.py

```diff
@@ -186,7 +186,9 @@ def _record_event(event, record_duplicates, user_id):
 
 
 @in_transaction
-def save_event_records(events, record_duplicates, return_identifiers):
+def save_event_records(
+    events, record_duplicates, return_identifiers, log_level=logging.INFO
+):
```

Comment:
what about just `debug=False` rather than setting the `log_level`

I'm also a fan of the global setting

```python
from anvil_labs.historic import set_debug_mode
set_debug_mode(True)
```

Typically I either want logging or I don't and having one place to set it feels nicer to me.

Reply:
I don't think I've been consistent. 

Atomic should use the Logger at some point. 

If we find a good pattern then we can amend other logging within other modules to match. 

I like `set_debug()` and `set_log_level()` as potential module level functions. The former for convenience. And the latter for more control.